docker.nix

```diff
@@ -0,0 +1,75 @@
+{ dockerTools
+, backend
+, frontend
+, busybox
+, cacert
+, caddy
+, iana_etc
+, writeScriptBin
+, writeTextFile }:
+
+let
+  inherit (dockerTools) buildLayeredImage;
+in
+{
+  backend-image = let
+    entry-point = writeScriptBin "entrypoint.sh" ''
+      #!/bin/sh
+      set -x
+      exec /bin/edna-server "$@"
+    '';
+  in buildLayeredImage {
+    name = "ghcr.io/serokell/edna-backend";
+    tag = "latest";
+
+    contents = [
+      backend
+      entry-point
+
+      iana_etc
+      busybox
+    ];
+
+    config = {
+      Entrypoint = "/bin/entrypoint.sh";
+      Cmd = [ "-c" "/config.yaml" ];
```

Comment:
Where does it come from? AFAIU this `/config.yaml` is a path inside the OCI image that we are creating here. In which place do we create this file in the image?

Reply:
You're expected to provide it externally via mount or volume.

To provide more context, `Cmd` are given as arguments to `Entrypoint`. When you `docker run foo:latest banana`, `banana` overrides the `Cmd` portion. So it's just a default.

Reply:
So AFAIU the relevant part of `edna.nix` (in another repo) is this:
```
        volumes = [
          "${configFile}:/config.yaml:ro"
        ];

        cmd = [
          "-c" "/config.yaml"
        ];
```

Reply:
Yes. I've decided to remove the default Cmd from this repo, since it depends on where the user mounts the config file anyway.